ydb/core/kqp/gateway/actors/scheme.h

```diff
@@ -86,10 +86,17 @@ class TSchemeOpRequestHandler: public TRequestHandlerBase<
                 {
                     LOG_DEBUG_S(ctx, NKikimrServices::KQP_GATEWAY, "Successful completion of scheme request"
                         << ", TxId: " << response.GetTxId());
-                        
-                    TResult result;
-                    result.SetSuccess();
-                    Promise.SetValue(std::move(result));
+
+                    if (!response.GetIssues().empty()) {
+                        NYql::TIssues issues;
+                        NYql::IssuesFromMessage(response.GetIssues(), issues);
+                        Promise.SetValue(NYql::NCommon::ResultFromIssues<TResult>(NYql::TIssuesIds::SUCCESS, "", issues));
```

Comment:
Should you put SUCCESS status here or maybe infer it from response.GetIssues()?

Reply:
Condition in case is already a status of response
This changes in ExecComplete status, so it means that result is success